src/Component.php

```diff
@@ -95,7 +90,7 @@ public function output($errors = null)
         $engine = app('view.engine.resolver')->resolve('blade');
         $engine->startLivewireRendering($this);
 
-        $view = $this->render();
+        $view = method_exists($this, 'render') ? app()->call([$this,'render']) : view("livewire.{$this->getName()}");
```

Comment:
Keep it readable like so
```php
$view = app()->call([$this, 'render']);
```

Reply:
i wanted to do that . but something in the tests blowed up :thinking: